src/common/helpers.js

```diff
@@ -0,0 +1,26 @@
+/**
```

Comment:
🎉 great that you got JSDoc working, I'll try to use it more!

Reply:
haha had to actually install an extension... 😢